supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -62,109 +62,446 @@ Introduce each release note with a heading that summarizes the release note. Thi
 * Do not expand abbreviations in headings. If you use an abbreviation in a heading, expand it on the first mention in the text below.
 * Avoid definitions in headings unless necessary for clarity. For example, use definitions to disambiguate different meanings of the same name: "The `journald` system role can tune the performance of the `journald` service".
 * Do not start the heading with a gerund. Use gerunds only for procedural content.
+[[jira-links-release-notes]]
+== Linking to Jira in release notes
 
-== Release note categories
+For customer reference, include links to Jira tickets on all _Known issues_  and _Fixed issues_ on the line directly after the entry. Do not place that link inside parenthesis or brackets. See examples later in this guidance.
```

Comment:
Added some more information so that products do not start to drift out of current practice. Needed some rephrasing, please feel free to adjust.

```suggestion
For customer reference, include links to Jira tickets on all _Known issues_  and _Fixed issues_. Some products provide ticket references for all release note types. Place the reference on the line directly after the entry, not inside parenthesis or brackets. See examples later in this guidance.
```